src/acceptor.rs

```diff
@@ -747,10 +747,11 @@ mod tests {
         }
         // Sort to ensure consistent comparison
         signature_schemes.sort();
-        assert_eq!(
-            &signature_schemes,
-            &[1025, 1027, 1281, 1283, 1537, 2052, 2053, 2054, 2055]
-        );
+        #[cfg(feature = "aws_lc_rs")] // aws-lc-rs includes P-521.
+        let expected_schemes = &[1025, 1027, 1281, 1283, 1537, 1539, 2052, 2053, 2054, 2055];
+        #[cfg(all(feature = "ring", not(feature = "aws_lc_rs")))]
+        let expected_schemes = &[1025, 1027, 1281, 1283, 1537, 2052, 2053, 2054, 2055];
```

Comment:
Preexisting, but I'd find 4 character hex (eg `0x0401`) more readable, and thats how iana documents them.

Reply:
I tacked on a separate commit to tidy this up: https://github.com/rustls/rustls-ffi/pull/441/commits/99728c05fff90d337b9814dc0e827619d4092577

I used the `rustls::SignatureScheme` enum instead of raw hex literals. I think that's the best option for readability and keeping everything in-sync with one source of truth.